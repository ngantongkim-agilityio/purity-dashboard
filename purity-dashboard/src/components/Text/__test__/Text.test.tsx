import { render } from '@testing-library/react';

// Components
import { Text } from '..';

describe('Text component', () => {
  it('Should render Text component with default props', () => {
    const { container } = render(<Text>Test</Text>);

    expect(container).toMatchSnapshot();
  });
});
