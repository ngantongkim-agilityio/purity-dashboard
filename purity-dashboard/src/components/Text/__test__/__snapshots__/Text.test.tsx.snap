// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component Should render Text component with default props 1`] = `
<div>
  <p
    class="text-primary-100 font-normal text-sm"
  >
    Test
  </p>
</div>
`;
